assets/js/stats.js

```diff
@@ -0,0 +1,64 @@
+const $ = require('jquery');
+global.$ = global.jQuery = $;
+
+import '../css/stats.css';
+
+import 'bootstrap';
+
+var flotConfig;
+var overviewSections;
+var style = getComputedStyle(document.body);
+var theme = {};
+
+theme.main = style.getPropertyValue('--color-main');
+theme.secondary = style.getPropertyValue('--color-menu');
+theme.dark = style.getPropertyValue('--color-headerMiddle');
+theme.light = style.getPropertyValue('--color-headerTop');
+
+$(document).ready(function() {
```

Comment:
Because its a rewrite should we use regular javascript instead of jQuery?

Reply:
These section is going away in the next ticket.